sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -215,9 +215,9 @@ statement
         routineCharacteristics
         RETURN (query | expression)                                    #createUserDefinedFunction
     | DROP TEMPORARY? FUNCTION (IF EXISTS)? identifierReference        #dropFunction
-    | DECLARE (OR REPLACE)? VARIABLE?
+    | DECLARE (OR REPLACE)? (VARIABLE | VAR)?
```

Comment:
shall we create a common parser rule for `VARIABLE | VAR`, like `namespace` did?

Reply:
sure, let me improve it.

Reply:
Instead of creating a common parser, using a lexer `VARIABLE: 'VARIABLE' | 'VAR';` shall be better

Reply:
What you mean by misuse is that the `developer` may have mistakenly used the keywords `VARIABLE` or `VAR` (directly use `VARIABLE` or `VAR` instead of `variable`)  in the file `SqlBaseParser.g4`, right?